crates/ef-testing/src/models/case.rs

```diff
@@ -65,6 +65,12 @@ impl BlockchainTestCase {
             )?;
             sequencer.setup_account(kakarot_account)?;
             sequencer.fund(address, account.balance)?;
+            if account.balance != U256::from(0)
```

Comment:
Do we want to put this behind a feature flag or do we want to duplicate the set_code_hash function in V1 ?
I would say the latter but maybe put it in default implementation of the Evm trait instead of duplication ?

Reply:
I would rather have the account code hash set in `KakarotAccount::new` in which you can also pass the balance so that you can trigger the condition has_code_or_nonce_or_balance to know whether the hash is h(bytecode) or h(empty) or 0

Reply:
It might be better indeed. But now the v1 will need to accept the balance parameter and not use it.

Reply:
done with [591ae5e](https://github.com/kkrt-labs/ef-tests/pull/731/commits/591ae5e90a598d12f075ab3a49d31e2cd4760763)